.metadata/Microsoft.Direct3D.DirectStorage.Manual.cs

```diff
@@ -0,0 +1,105 @@
+// Missing constants from https://learn.microsoft.com/en-us/windows/win32/dstorage/dstorage-constants
```

Comment:
Since we manually copy now the errors in this CS file and then transform them via bindgen anyhow, isn't it easier to just define these constants (they seem to all be error related) in their own module, like the first version of this library?

Reply:
This allows us to generate a more consistent view of `.winmd`.

I learned how to use `enums.json` which should be able to parse these `#define`s, but it seems to revolve around adding them to an ever-growing list of an enum type, which does not work for these values of type `HRESULT`.

I'd like to see if that can be addressed upstream (or if there is already an existing solution for this) to emit new loose constants with the `HRESULT` type.

Reply:
I see. Can we on the other hand output the created constants on the Rust side into their own module automatically, so that they get better grouped?

Reply:
The only quick and dirty way I can think of is by using `--filter` and running `--bindgen` twice, but I don't know if it supports negation.

Alternatively we can move these manual constants to an `Errors` module inside `DirectStorage` (that better matches the layout of the headers) via the metadata.

Reply:
For future reference: found the bug why `#define` was not being parsed from `dstorageerr.h`: https://github.com/Tsukisoft/direct-storage-rs/pull/5. This whole file can be removed again 🎉.